directories/src/services/iota.js

```diff
@@ -0,0 +1,140 @@
+import IOTA from 'iota.lib.js';
+import { IOTA_API_PROVIDER } from '../config';
+import curl from 'curl.lib.js';
+
+export const iota = new IOTA();
+
+export const providerIota = new IOTA({
+  provider: IOTA_API_PROVIDER
+});
+
+curl.init();
+const MAX_TIMESTAMP_VALUE = (Math.pow(3,27) - 1) / 2;
+
+export const prepareTransfers = (data) => {
+  return new Promise((resolve, reject) => {
+    providerIota.api.prepareTransfers(
+      data.seed,
+      [{
+        'address': data.address,
+        'value': data.value,
+        'message': data.message,
+        'tag': data.tag
+      }],
+      (error, prepareTransfers) => {
+        if(error) {
+          console.log('IOTA prepareTransfers error ', error);
+        } else {
+          console.log('IOTA prepareTransfers data ', prepareTransfers);
+          resolve(prepareTransfers);
+        }
+      }
+    )
+  })
+};
+
+const attachToTangleCurl = (trunkTransaction, branchTransaction, minWeightMagnitude, trytes, callback) => {
```

Comment:
You may be able to use this: 
 https://github.com/iotaledger/curl.lib.js/blob/2da1de36123d89b0c7dcababe24f72828bc5f794/src/curl.lib.js#L41